__mocks__/permissions-android.ts

```diff
@@ -0,0 +1,38 @@
+import { Permission, PermissionStatus } from 'react-native'
+
+const androidPermissions: { [key in Permission]: PermissionStatus } = {
```

Comment:
Tu peux utiliser la case des constantes :
```suggestion
const INITIAL_ANDROID_PERMISSIONS: { [key in Permission]: PermissionStatus } = {
```